source/code/source/playfab/PlayFabWinHttpPlugin.cpp

```diff
@@ -466,6 +469,27 @@ namespace PlayFab
             return activeRequestCount;
         }
     }
+
+    void PlayFabWinHttpPlugin::CloseOutConnections(std::unique_ptr<CallRequestContainer> requestContainer, HINTERNET hRequest, HINTERNET hConnect, HINTERNET hSession)
```

Comment:
nit: the name `CloseOutConnections` feels like it's missing the really important part of what this function does. Handling the callback is what's most important for logically following this code. what about calling this `CompleteRequest` to mirror the `ExecuteRequest where it's called.